packages/vscode-extension/test/extension/handlers.test.ts

```diff
@@ -1699,10 +1696,10 @@ describe("handlers", () => {
       const createProgressBar = sandbox
         .stub(vsc_ui.VS_CODE_UI, "createProgressBar")
         .returns(progressHandler);
-      sandbox.stub(globalVariables, "core").value(new MockCore());
-      sandbox.stub(vscode.commands, "executeCommand");
-      sandbox.stub(globalState, "globalStateUpdate");
-      const getApp = sandbox.stub(AppStudioClient, "getApp").throws("error");
+      sinon.stub(globalVariables, "core").value(new MockCore());
```

Comment:
We should avoid using `sinon.stub`, is there any reason not using `sandbox.stub`?

Reply:
Copy from dev when merging PR, I will fix this in another PR.